sf-docker/all-in-one/create-wl-stream-generation.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+source .env
```

Comment:
this did not work on test server

Reply:
Confirmed, fixed now with a bash function for getting the values from .env file.